java/client/src/main/java/glide/api/models/commands/json/JsonGetOptions.java

```diff
@@ -0,0 +1,56 @@
+/** Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.models.commands.json;
+
+import glide.api.commands.servermodules.GlideJson;
+import java.util.ArrayList;
+import java.util.List;
+import lombok.Builder;
+import lombok.Getter;
+
+/** Additional parameters for {@link GlideJson#get} command. */
+@Getter
+@Builder
+public final class JsonGetOptions {
+    /** ValKey API string to designate INDENT */
+    public static final String INDENT_VALKEY_API = "INDENT";
```

Comment:
Valkey API is repeated a number of times in the variable names and comments. This can be removed.

Would recommend moving these constants to a static class file called servermodules/constants/JsonGetKeywords.java or a common JsonKeywords.java.

Reply:
Probably not the best, but this approach has been around for a while. See `LPosOptions` and `RestoreOptions` for more `XXX_VALKEY_API`.